environment.yml

```diff
@@ -9,5 +9,4 @@ dependencies:
   - pytest=6.2.4
   - pytest-cov=2.12.1
   - mypy=0.910
-  - data-science-types=0.2.23
-  - pytype=2020.10.8
\ No newline at end of file
+  - data-science-types=0.2.23
```

Comment:
While not necessary to run the project, could still include pytype from [pip in conda environment file](https://stackoverflow.com/a/35245610)